liblangutil/ParserBase.h

```diff
@@ -36,7 +36,13 @@ class Scanner;
 class ParserBase
 {
 public:
-	explicit ParserBase(ErrorReporter& errorReporter): m_errorReporter(errorReporter) {}
+	/// Set @_parserErrorRecover to true  for additional Steve-Johnson-style error recovery.
```

Comment:
Perhaps just "error recovery" since we might diverge from that style or even haven't fully implemented it?